tests/contrib/langchain/test_langchain_llmobs.py

```diff
@@ -147,6 +147,15 @@ def _embed_documents(cls, embedding_model, documents, mock_tracer, cassette_name
         LLMObs.disable()
         return mock_tracer.pop_traces()[0]
 
+    @classmethod
+    def _invoke_tool(cls, tool, tool_input, mock_tracer, cassette_name):
+        LLMObs.enable(ml_app=cls.ml_app, integrations_enabled=False, _tracer=mock_tracer)
+        with get_request_vcr(subdirectory_name=cls.cassette_subdirectory_name).use_cassette(cassette_name):
+            if LANGCHAIN_VERSION > (0, 1):
```

Comment:
Is there any particular reason we're version gating this here?

Reply:
> Note, you'll only need to trace BaseTool.invoke() for langchain V1 and up.
In the ticket description you mentioned this so I was trying to make sure it's the case. I'm guessing since it's in langchain_community test suite it's always going to be the case ?